tools/dev_setup.sh

```diff
@@ -19,10 +19,37 @@ EOD
 
 vagrant up
 
+# fix permissions
+cat <<EOD | vagrant ssh
+sudo chown vagrant /home/vagrant/go
+sudo chown vagrant /home/vagrant/go/src/
+sudo chown vagrant /home/vagrant/go/src/github.com
+EOD
+
+# golang setup
+cat <<EOD | vagrant ssh
+mkdir -p /opt
+wget -q https://dl.google.com/go/go1.11.2.linux-amd64.tar.gz
+sudo tar -xzf go1.11.2.linux-amd64.tar.gz -C /opt
```

Comment:
This is different from what we build against for staging and prod (https://github.com/DataDog/datadog-process-agent/blob/master/packaging/build_ddagent_release.sh#L27). Maybe we should keep these in sync so we're always testing with the same version in dev as we will release?